config/overlays/e2e-release-0.3/ironic.env

```diff
@@ -0,0 +1,10 @@
+HTTP_PORT=6180
+PROVISIONING_IP=192.168.222.199
+DEPLOY_KERNEL_URL=http://192.168.222.199:6180/images/ironic-python-agent.kernel
+DEPLOY_RAMDISK_URL=http://192.168.222.199:6180/images/ironic-python-agent.initramfs
+IRONIC_ENDPOINT=https://192.168.222.199:6385/v1/
+IRONIC_INSPECTOR_ENDPOINT=https://192.168.222.199:5050/v1/
+CACHEURL=http://192.168.222.199/images
+IRONIC_FAST_TRACK=true
+IRONIC_KERNEL_PARAMS=console=ttyS0
+IRONIC_INSPECTOR_VLAN_INTERFACES=all
```

Comment:
Will this consume ironic-image from main? If so, you need to set `USE_IRONIC_INSPECTOR=true`.

Reply:
Yes, we use the same ironic image for all jobs currently and that is the latest. Please add it @mquhuy

Reply:
Wait a second, this is for BMO. In #1544 you did not touch those, do we need to?
- BMO: https://github.com/metal3-io/baremetal-operator/blob/main/config/overlays/e2e/ironic.env
- Ironic: https://github.com/metal3-io/baremetal-operator/blob/main/ironic-deployment/overlays/e2e/ironic_bmo_configmap.env

Does BMO need to have the variable or is it only relevant for Ironic?

Reply:
BMO requires Inspector before 0.5.0. Does it answer your question?